.studio/golang

```diff
@@ -260,6 +260,53 @@ function go_component_static_tests() {
 # Adding auto tab complete
 complete -F _component_auto_complete go_component_static_tests
 
+document "go_component_lint" <<DOC
+  Runs the golangci linter for a component
+DOC
+function go_component_lint() {
+  [ "x$1" == "x" ] && error "Missing component name argument; try 'describe ${FUNCNAME[0]}'" && return 1
```

Comment:
[nit] except that `describe "go_component_lint"` doesn't exactly tell me what the positional argument is supposed to be. 😉